scripts/deploy/version_bump_pr_steps.rb

```diff
@@ -48,25 +48,19 @@ def create_version_bump_pr()
 
     pr_description = create_pr_description()
 
-    options = {}
-    if (@is_dry_run)
-        options["draft"] = true
-    end
-
     response = github_login.create_pull_request(
         "stripe/stripe-android",
         "master",
         release_branch,
         "Bump version to #{@version}",
         pr_description,
-        options
+        draft: @is_dry_run,
     )
 
-    rputs "Merge the version bump PR"
-
     pr_url = response.html_url
-    puts "Opening url #{pr_url}"
-    `open #{pr_url}`
+    open_url(pr_url)
+
+    rputs "Merge the version bump PR"
```

Comment:
Should we say, merge then press enter once it's merged?

Reply:
This is the output of the script at this point:
```
Opening url https://github.com/stripe/stripe-android/pull/8887
Merge the version bump PR
Press enter to continue...
```

The `wait_for_user` function outputs "Press enter to continue...". I think this is sufficient but lmk if you think I should update this!